app/models/concerns/has_rdvs_and_participations.rb

```diff
@@ -1,6 +1,14 @@
-module HasRdvs
+module HasRdvsAndParticipations
```

Comment:
`HasParticipationsToRdvs` ?

Reply:
ce sont des modèles où l'on peut faire `model.rdvs` et `model.participations`, donc `HasRdvsAndParticipations` me paraît plus clair

Reply:
Oui mais les rdvs découlent des participations, d'où  ma suggestion.
C'est juste que généralement avoir `and` dans le nom d'un module ou d'une classe est un code smell, qui signifie que tu devrais avoir deux modules/classes séparées, mais ici tu veux les avoir ensemble parce que justement les 2 sont liés, et donc j'ai pensé que le nom pourrait le refléter.
Mais c'est un détail 🤷‍♂️ .